composer.json

```diff
@@ -17,11 +17,16 @@
         "friendsofphp/php-cs-fixer": "^2.0",
         "phpunit/phpunit": "^9.5",
         "symfony/phpunit-bridge": "^3.0|^4.0"
+        "phpstan/phpstan": "^1.8"
     },
     "autoload": {
         "psr-4": { "Ivory\\OrderedForm\\": "src/" }
     },
     "autoload-dev": {
         "psr-4": { "Ivory\\Tests\\OrderedForm\\": "tests/" }
+    },
+    "scripts": {
+        "phpstan": "phpstan analyse --level=5 src",
```

Comment:
do not we want the `tests` folder to be checked as well? :innocent:

Reply:
It would be nice, but it opened a not pretty rabbit hole... It's necessary to annotate many variables and with more than one class (getPosition and setPosition are in different interfaces...).

I can give it a shot again